src/info_typer/events.clj

```diff
@@ -0,0 +1,72 @@
+(ns info-typer.events
+  (:require [info-typer.amqp             :as amqp]
+            [clojure.tools.logging       :as log]
+            [langohr.core                :as rmq]
+            [langohr.basic               :as lb]
+            [langohr.channel             :as lch]
+            [langohr.exchange            :as le]
+            [langohr.queue               :as lq]
+            [langohr.consumers           :as lc]
+            [info-typer.config           :as cfg]
+            [service-logging.thread-context :as tc]
+            [clojure-commons.error-codes :as ce])
+  (:import [org.cyverse.events.ping PingMessages
+                                    PingMessages$Ping
+                                    PingMessages$Pong]
+           [com.google.protobuf.util JsonFormat]))
+
+(def pong-key "events.info-typer.pong")
+
+(defn ping-handler
+  [channel metadata payload]
+  (lb/publish channel (cfg/amqp-events-exchange) pong-key
```

Comment:
When publishing, we should create a header which identifies the publisher. In this case, the service. This is to help with debugging in the future, in the case that a particular topic/routing-key is published from two different services.